### PR TITLE
Properly load default language entries from mods and Minecraft.

### PR DESCRIPTION
**Pull request by LambdAurora**
This fixes dedicated servers not having any language entries loaded.

### CODE REVIEW

**Review by EnnuiL — APPROVED** [review]
Awesome!

**Review by Leo40Git — APPROVED** [review]
Y E S

**Review by violet-sh — APPROVED** [review]
Yesssss

**Comment by Gaming32** [comment]
ComputerCraft has [an equivalent mixin](https://github.com/cc-tweaked/cc-restitched/blob/mc-1.19.x/stable/src/main/java/dan200/computercraft/fabric/mixin/MixinLanguage.java). What would you recommend they do? https://github.com/cc-tweaked/cc-restitched/issues/133

**Comment by LambdAurora** [comment]
> ComputerCraft has [an equivalent mixin](https://github.com/cc-tweaked/cc-restitched/blob/mc-1.19.x/stable/src/main/java/dan200/computercraft/fabric/mixin/MixinLanguage.java). What would you recommend they do? [cc-tweaked/cc-restitched#133](https://github.com/cc-tweaked/cc-restitched/issues/133)

Fabric API recently got a similar mixin, though I take a wildly different approach.

I think dropping the mixin there would make sense.

File: library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/mixin/server/LanguageMixin.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+@DedicatedServerOnly
+@Mixin(Language.class)
+public class LanguageMixin {
+	@Redirect(
+			method = "create",
+			at = @At(value = "INVOKE", target = "Lcom/google/common/collect/ImmutableMap$Builder;build()Lcom/google/common/collect/ImmutableMap;")
+	)
+	private static ImmutableMap<String, String> create(ImmutableMap.Builder<String, String> cir) {
+		var map = new Object2ObjectOpenHashMap<>(cir.buildOrThrow());
```
Comment (OroArmor) [inline library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/mixin/server/LanguageMixin.java]:
```suggestion
	private static ImmutableMap<String, String> create(ImmutableMap.Builder<String, String> builder) {
		var map = new Object2ObjectOpenHashMap<>(builder.buildOrThrow());
```

**Review by OroArmor — CHANGES_REQUESTED** [review]
One nitpick

**Review by OroArmor — APPROVED** [review]

**Pull request merged**